### PR TITLE
Release 2.0.0

### PR DESCRIPTION
**Pull request by crow**
Release 2.0.0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,13 @@
+Version 2.0.1 - January 10, 2020
+================================
+- Renamed plugin package name from airship to airship_flutter
+- Updated iOS SDK to 13.0.4
+- Updated Android SDK to 12.1.0
+- Implemented attributes functionality in iOS and Android
+
```
Comment (BrianBatchelder) [inline CHANGELOG.md]: How come no 2.0.0?

Comment (crow) [reply CHANGELOG.md]: wtf happened, I think I messed this up scripting or something

Comment (crow) [reply CHANGELOG.md]: good catch

**Comment by marc-scig** [comment]
👌

**Comment by BrianBatchelder** [comment]
I'm still not seeing the actual version numbers (in the code). Are they in a previous PR?

**Comment by crow** [comment]
the pubspec.yaml, etc are all already updated to 2.0.0 - this was done in previous prs

**Pull request merged**